### ISSUE TITLE
autoupdate prompt not responding

### ISSUE THREAD

**Issue by RainyNyte** [TeamREPENTOGON/REPENTOGON#239]
the window asking about autoupdates freezes and stops responding, first time trying to use repentagon and I temporarily removed all other mods to see if there were just too many and it was slowing the process down. I don't have a dsound.ini file because it never gets far enough to generate it

**Comment by sickofitalll** [comment]
yup same here i cant get past the window that asks me about the autoupdates, its just frozen.

**Comment by RainyNyte** [comment]
Ended up figuring out a fix, make a dsound.ini file in the same folder you installed the mod to, and put "CheckForUpdates = 0" as the only line in it, if that doesn't work then copy it to your games save directory as well so that the file exists in both folders

**Comment by Sylmir** [comment]
Self-note: investigate mINI (possibly throwing an exception ?)

**Comment by jerbmega** [comment]
Closing - no longer valid with the launcher migration.
